recipes/openff-toolkit/conda_build_config.yaml

```
@@ -0,0 +1,2 @@
+channel_targets:
+- conda-forge openff_toolkit_rc
```

This should reflect the exact name of the package:

```suggestion
- conda-forge openff-toolkit_rc
```